model-optimizer/extensions/ops/range.py

```diff
@@ -48,14 +48,19 @@ def __init__(self, graph: Graph, attrs: dict):
             'type': self.op,
             'op': self.op,
 
-            'version': 'opset1',
+            'version': 'opset4',
             'infer': self.infer,
+            'type_infer': self.type_infer,
 
             'in_ports_count': 3,
             'out_ports_count': 1,
         }
         super().__init__(graph, mandatory_props, attrs)
 
+    @staticmethod
+    def type_infer(node: Node):
+        node.out_port(0).set_data_type(node['output_type'])
```

Comment:
The function will fail for Range-1 because it does not have `output_type` attribute.

So for the Range-1 the type infer function should be different and propagate 0-input tensor's type

Reply:
`output_type` will be created for `opset1` op, but won't serialize to IR because of `backend_attrs`